### ISSUE TITLE
[Phase 1][YSQL][colocation] Wrong table description for tablegroups and colocated tables

### ISSUE THREAD

**Issue by yifanguan** [yugabyte/yugabyte-db#12186]
Jira Link: [[DB-381]](https://yugabyte.atlassian.net/browse/DB-381)
### Description

```
CREATE DATABASE colocation_test colocated = true;
\c colocation_test
CREATE TABLE tbl (h1 int, h2 int, r1 int, r2 int);
CREATE INDEX idx on tbl (h1 hash);
CREATE INDEX idx2 on tbl ((h1, h2) hash);
\d tbl
                Table "public.tbl"
 Column |  Type   | Collation | Nullable | Default 
--------+---------+-----------+----------+---------
 h1     | integer |           |          | 
 h2     | integer |           |          | 
 r1     | integer |           |          | 
 r2     | integer |           |          | 
Indexes:
    "idx" lsm (h1 HASH)
    "idx2" lsm ((h1, h2) HASH)

\c yugabyte
DROP DATABASE colocation_test;

```

On the other hand, localhost:7000 shows:
for `colocation_test.idx`
```
h1            | 10 | int32 NOT NULL NOT A PARTITION KEY
ybidxbasectid | 11 | binary NOT NULL NOT A PARTITION KEY
```

and

for `colocation_test.idx2`
```
h1            | 10 | int32 NOT NULL NOT A PARTITION KEY
h2            | 11 | int32 NOT NULL NOT A PARTITION KEY
ybidxbasectid | 12 | binary NOT NULL NOT A PARTITION KEY
```

[DB-381]: https://yugabyte.atlassian.net/browse/DB-381?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by yifanguan** [comment]
We should check for tablegroup as well.

**Comment by yifanguan** [comment]
Issue resolved by commit: 2ad388f
